presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1113,6 +1114,15 @@ public Void visitTableWriter(TableWriterNode node, Integer indent)
             return processChildren(node, indent + 1);
         }
 
+        @Override
+        public Void visitAnalyzeFinish(AnalyzeFinishNode node, Integer indent)
+        {
+            print(indent, "- AnalyzeCommit[%s] => [%s]", node.getTarget(), formatOutputs(node.getOutputSymbols()));
```

Comment:
`AnalyzeCommit` or `AnalyzeFinish`?

Reply:
Same thing. For some reason `TableFinish` is displayed as `TableCommit`. This was done for consistency.